nms/app/views/equipment/__tests__/EquipmentGatewayPoolTest.js

```diff
@@ -243,19 +250,34 @@ describe('<GatewayPools />', () => {
 });
 
 describe('<AddEditGatewayPoolButton />', () => {
+  let lteNetworkIdGatewayPoolsGatewayPoolIdGet;
+
   afterEach(() => {
-    MagmaAPIBindings.postLteByNetworkIdGatewayPools.mockClear();
-    MagmaAPIBindings.putLteByNetworkIdGatewaysByGatewayIdCellularPooling.mockClear();
+    MagmaAPI.lteNetworks.lteNetworkIdGatewayPoolsPost.mockClear();
+    MagmaAPI.lteGateways.lteNetworkIdGatewaysGatewayIdCellularPoolingPut.mockClear();
```

Comment:
Does this conflict with https://github.com/magma/magma/pull/12718?

Reply:
Yes it is merged by now